redis/asyncio/parser.py

```diff
@@ -50,19 +50,19 @@ async def get_keys(self, *args: Any) -> Optional[Tuple[str, ...]]:
             return None
 
         try:
-            command = self.commands[args[0]]
+            command = self.commands[args[0].upper()]
```

Comment:
Since most users won't use raw commands, we should only keep this in the except block. Please remove this change.